plotter/bubbles_test.go

```diff
@@ -66,29 +42,61 @@ func ExampleBubbles() {
 	}
 
 	n := 10
-	bubbleData := randomTriples(n)
+	scatterData := randomTriples(n)
+
+	// Calculate the range of Z values.
+	minZ, maxZ := math.Inf(1), math.Inf(-1)
+	for _, xyz := range scatterData {
+		if xyz.Z > maxZ {
+			maxZ = xyz.Z
+		}
+		if xyz.Z < minZ {
+			minZ = xyz.Z
+		}
+	}
 
 	p, err := plot.New()
 	if err != nil {
 		log.Panic(err)
 	}
-	p.Title.Text = "Bubbles"
+	p.Title.Text = "Bubble Plot"
```

Comment:
If this is changed to "Bubbles", then the reference image should hopefully not change